net/xtables-wgobfs/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (C) 2022 Wei Chen <weichen302@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=xtables-wgobfs
+PKG_VERSION:=0.4.2
+PKG_RELEASE:=1
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/xt_wgobfs-$(PKG_VERSION)
+PKG_SOURCE:=xt_wgobfs-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:= https://github.com/infinet/xt_wgobfs/releases/download/v$(PKG_VERSION)/
+PKG_HASH:=09fa493d8305e1fa3224a940cab607b1860a9b5d9d395615105c7009e2bec767
+
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_BUILD_DEPENDS:=iptables
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+PKG_MAINTAINER:=Wei Chen <weichen302@gmail.com>
+
+include $(INCLUDE_DIR)/kernel.mk
+include $(INCLUDE_DIR)/package.mk
+
+define Package/xtables-wgobfs
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Firewall
+	TITLE:=iptables WireGuard obfuscation extension
+	URL:=https://github.com/infinet/xt_wgobfs
+	MAINTAINER:=Wei Chen <weichen302@gmail.com>
+	DEPENDS:= +iptables +kmod-ipt-core +libxtables
+endef
+
+define Package/xtables-wgobfs/description
+	An iptables extension for WireGuard obfuscation
+endef
+
+# uses GNU configure
+CONFIGURE_ARGS+= \
+	--with-kbuild="$(LINUX_DIR)" \
+	--with-xtlibdir="/usr/lib/iptables" \
+
+define Build/Compile
+	+$(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR) \
+		ARCH="$(LINUX_KARCH)" \
+		CROSS_COMPILE="$(TARGET_CROSS)" \
+		DESTDIR="$(PKG_INSTALL_DIR)" \
+		DEPMOD="/bin/true" \
+		all
+endef
+
+define Build/Install
+	mkdir -p $(PKG_INSTALL_DIR)/usr/lib/iptables/
+	$(MAKE) -C $(PKG_BUILD_DIR) \
+		ARCH="$(LINUX_KARCH)" \
+		CROSS_COMPILE="$(TARGET_CROSS)" \
+		DESTDIR="$(PKG_INSTALL_DIR)" \
+		DEPMOD="/bin/true" \
+		install
+endef
+
+define Package/iptables-mod-wgobfs
+	$(call Package/xtables-wgobfs)
```

Comment:
Since `Package/xtables-wgobfs` is only used once (here), you can set the values here directly.

Reply:
Thank you for the review! I have made some changes accordingly.